server/src/main/java/com/hedera/block/server/Server.java

```diff
@@ -1,38 +1,96 @@
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
 package com.hedera.block.server;
 
-import com.hedera.block.protos.EchoServiceGrpcProto;
+import com.hedera.block.protos.BlockStreamServiceGrpcProto;
+import com.hedera.block.server.mediator.LiveStreamMediatorImpl;
+import com.hedera.block.server.persistence.WriteThroughCacheHandler;
+import com.hedera.block.server.persistence.cache.BlockCache;
+import com.hedera.block.server.persistence.cache.LRUCache;
+import com.hedera.block.server.persistence.storage.BlockStorage;
+import com.hedera.block.server.persistence.storage.FileSystemBlockStorage;
+import io.grpc.stub.ServerCalls;
 import io.grpc.stub.StreamObserver;
+import io.helidon.config.Config;
 import io.helidon.webserver.WebServer;
 import io.helidon.webserver.grpc.GrpcRouting;
 import io.helidon.webserver.http.HttpRouting;
 
+import java.io.IOException;
+import java.util.logging.Logger;
+import java.util.stream.Stream;
+
+import static com.hedera.block.server.Constants.*;
+
 /**
  * Main class for the block node server
  */
 public class Server {
-    private Server() {
-        // Not meant to be instantiated
-    }
+
+    // Function stubs to satisfy the bidi routing param signatures.  The implementations are in the service class.
+    private static ServerCalls.BidiStreamingMethod<Stream<BlockStreamServiceGrpcProto.Block>, StreamObserver<BlockStreamServiceGrpcProto.Block>> clientBidiStreamingMethod;
+    private static ServerCalls.BidiStreamingMethod<Stream<BlockStreamServiceGrpcProto.BlockResponse>, StreamObserver<BlockStreamServiceGrpcProto.Block>> serverBidiStreamingMethod;
+
+    private static final Logger logger = Logger.getLogger(Server.class.getName());
+
+    private Server() {}
 
     /**
      * Main entrypoint for the block node server
      *
      * @param args Command line arguments. Not used at present,
      */
     public static void main(String[] args) {
-        WebServer.builder()
-                .port(8080)
-                .addRouting(HttpRouting.builder()
-                        .get("/greet", (req, res) -> res.send("Hello World!")))
-                .addRouting(GrpcRouting.builder()
-                        .service(new EchoService())
-                        .unary(EchoServiceGrpcProto.getDescriptor(),
-                                "EchoService",
-                                "Echo",
-                                Server::grpcEcho))
-                .build()
-                .start();
-    }
 
-    static void grpcEcho(EchoServiceGrpcProto.EchoRequest request, StreamObserver<EchoServiceGrpcProto.EchoResponse> responseObserver) {}
+        try {
+
+            // Set the global configuration
+            Config config = Config.create();
```

Comment:
```suggestion
            final Config config = Config.create();
```
We prefer to use final liberally, as many places as possible. I wish Java was final by default, like Rust is. We find it helpful when reading code that at the line we encounter the variable declaration, we can know its possible mutability as we read further code. Knowing it is final is one less thing we have to keep in our head as we read code (because almost everything turns out to be final).

Also feel free to use `final var config = Config.create();`. I'm a big fan of `var`, other people hate it, so either way is fine.

This comment applies to all local variables.

Reply:
fixed